server/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM python:3.11
+
+COPY server.py /code/
+
+WORKDIR /code
+
+RUN pip install flask
+
+ENTRYPOINT [ "python" ]
+
+CMD [ "klaatu_server.py" ]
```

Comment:
Q: What is "klaatu_server.py"? Or should this be "server.py" (since that's what we `COPY`d on L3 and changed our `WORKDIR` to on L5.

Reply:
Good catch!